src/chart/onResize.js

```diff
@@ -164,15 +173,17 @@ export default function onResize() {
             this.config.formDescription_col
           ]
       );
-  if (this.config.y.column === "Field" && this.config.fieldDescription_col)
+  if (this.config.y.column === "Form: Field" && this.config.fieldDescription_col)
```

Comment:
This isn't giving a title for me. Also, it should probably be: `this.config.y.column === "Form: Field" && (this.config.fieldDescription_col || this.config.formDescription_col)`